src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -1,7 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- The documentation in here must be kept up to date with that embedded in comments in newrelic.config -->
-<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="urn:newrelic-config" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nr="Configuration.xsd">
-  <xs:element name="configuration">
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:tns="urn:newrelic-config" targetNamespace="urn:newrelic-config" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nr="Configuration.xsd">
+
+    <xs:complexType name="ErrorClassType">
```

Comment:
Should we consider a different type name here? How about `ErrorClassCollection` to more closely represent that this type represents a collection of error classes?